src/Microsoft.ML.Transforms/UngroupTransform.cs

```diff
@@ -337,7 +367,7 @@ public void Save(ModelSaveContext ctx)
             /// </summary>
             public bool[] GetActiveInput(Func<int, bool> predicate)
             {
-                var activeInput = Utils.BuildArray(ColumnCount, predicate);
+                var activeInput = Utils.BuildArray(OutputSchema.Count, predicate);
```

Comment:
>OutputSchema [](start = 51, length = 12)

_inputSchema.Count ?

Reply:
`OutputSchema.Count` is the same to `_inputSchema.Count` but I will change it to `_inputSchema.Count`.

---
In reply to: [246128093](https://github.com/dotnet/machinelearning/pull/2066#discussion_r246128093) [](ancestors = 246128093)